common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -271,3 +288,163 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
         self.assertFalse(self.discussion_management_page.is_category_selected())
+
+
+@attr(shard=6)
+class DivisionSchemeTest(BaseDividedDiscussionTest, BaseDiscussionMixin):
+    """
+    Tests for changing the division scheme for Discussions.
+    """
+
+    def add_enrollment_modes(self, modes):
+        """
+        Adds enrollment modes to the course, and then goes to the
+        discussion tab on the instructor dashboard.
+        """
+        add_enrollment_course_modes(self.browser, self.course_id, modes)
+
+        self.view_discussion_management_page()
+
+    def view_discussion_management_page(self):
+        """
+        Go to the discussion tab on the instructor dashboard.
+        """
+        self.instructor_dashboard_page.visit()
+        self.instructor_dashboard_page.select_discussion_management()
+        self.discussion_management_page.wait_for_page()
+
+    def setup_thread_page(self, thread_id):
+        """
+        This is called by BaseDiscussionMixin.setup_thread.
+        """
+        self.thread_page = DiscussionTabSingleThreadPage(
+            self.browser, self.course_id, self.discussion_id, thread_id
+        )
+        self.thread_page.visit()
+
+    def test_not_divided_hides_discussion_topics(self):
+        """
+        Tests that discussion topics are hidden iff discussion division is disabled.
+        """
+        # Initially "Cohort" is the selected scheme.
+        self.assertTrue(
+            self.discussion_management_page.division_scheme_visible(self.discussion_management_page.COHORT_SCHEME)
+        )
+        self.assertEqual(
+            self.discussion_management_page.COHORT_SCHEME,
+            self.discussion_management_page.get_selected_scheme()
+        )
+        self.check_discussion_topic_visibility(visible=True)
+
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.NOT_DIVIDED_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=False)
+
+        # Reload the page and make sure that the change was persisted
+        self.reload_page(topics_visible=False)
+        self.assertTrue(self.discussion_management_page.division_scheme_visible(
+            self.discussion_management_page.COHORT_SCHEME)
+        )
+        self.assertEqual(
+            self.discussion_management_page.NOT_DIVIDED_SCHEME,
+            self.discussion_management_page.get_selected_scheme()
+        )
+
+        # Select "cohort" again and make sure that the discussion topics appear.
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.COHORT_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=True)
+
+    def test_disabling_cohorts(self):
+        """
+        Test that disabling cohorts hides the cohort division scheme iff it is not the selected scheme
```

Comment:
After making these comments I remembered that it was a common shorthand for if and only iff.